module/actor.js

```diff
@@ -1014,6 +1025,24 @@ class DCCActor extends Actor {
       })
     }
 
+    // Add Strength or Agility modifier to attack rolls
+    let modifier;
```

Comment:
It would be nice to integrate the attack bonus and/or deed die into this setting, which should be doable by adding an @ab term for all non-zero level characters.